README.md

```diff
@@ -23,11 +23,14 @@ What to adjust:
     <img src="/images/logo.png" alt="Logo" height="200">
   </a>
 
-<h3 align="center">DigiWF <i>repo or project name</i></h3>
+<h3 align="center">DigiWF E-Mail Integration</h3>
 
   <p align="center">
-    <i>Add a here a short description</i>
-    <br /><a href="#">Report Bug</a>
+    This is a Spring Boot Starter library to send e-mails in the DigiWF environment.
+    It can be used to send mails asynchronously through kafka.<br>For attachments to not clutter the EventBus,
```

Comment:
For this and all following comments regarding Kafka / Event Bus:
Kafka is already implemented. The binder is chosen in the mail-integration-client artifact, because that's what we need in our environment. That's why I worded it that way (e. g. "Kafka EventBus" instead of just "EventBus")

Reply:
Done